### ISSUE TITLE
Update minimum Bitcoin Core version requirement after SW activates

### ISSUE THREAD

**Issue by unsystemizer** [JoinMarket-Org/joinmarket#746]
Soon (after SW activates), the minimum Bitcoin Core req may need to updated at:
https://github.com/JoinMarket-Org/joinmarket/wiki/Running-JoinMarket-with-Bitcoin-Core-full-node

"Any flavor of up-to-date bitcoin client full node software. Bitcoin Core version 0.10 or above. "

**Comment by AdamISZ** [comment]
Good point, albeit SW is not in this code version yet. That line could be edited various ways, we don't support anything except Bitcoin Core (in case RPC is different), and one could argue 0.13.1 as minimum, or one could argue 0.10 is still OK after soft fork.
